godtools/App/Features/AppLanguage/Presentation/DownloadableLanguages/ViewModel/DownloadableLanguagesViewModel.swift

```diff
@@ -42,6 +43,12 @@ class DownloadableLanguagesViewModel: ObservableObject {
         self.downloadToolLanguageUseCase = downloadToolLanguageUseCase
         self.removeDownloadedToolLanguageUseCase = removeDownloadedToolLanguageUseCase
         
+        Self.languageDownloadHandler.setDelegate(self)
+        Self.languageDownloadHandler
+            .getActiveDownloadsObserver()
+            .receive(on: DispatchQueue.main)
+            .assign(to: &$activeDownloads)
```

Comment:
Hey @rachaelblue do you know if this is keeping a strong reference since using ```assign``` on a ```static``` reference?
Does the ViewModel get deallocated?  I'll try running this PR soon to see.
Also my original thought was that the progress would get updated through the delegate handler.  Was that causing issues though with Double progress?

Reply:
Just checked by tapping a few languages to download and when I navigated back I did see the deinit log for the DownloadableLanguagesViewModel so that's good to know!